ManageWiki.php

```diff
@@ -2115,15 +2115,15 @@
 		'restricted' => false,
 		'type' => 'check',
 		'overridedefault' => false,
-		'help' => '',
+		'help' => 'This setting allows anonymous users to change the language for them selfs.',
```

Comment:
"Enabling allowing anonymous users to control the language they view the wiki in"

Reply:
Thanks, done!